src/plugins/utils/http-utils.ts

```diff
@@ -26,6 +27,7 @@ export type HttpPluginConfig = {
     logicalServiceName: string;
     urlsToInclude: RegExp[];
     urlsToExclude: RegExp[];
+    urlsToTrace: RegExp[];
```

Comment:
suggestion: Instead of creating a new config option, extend `addXRayTraceIdHeader` to be either a boolean (current) or a regex array (proposed extension). When it is a regex array, it would only add headers to matching URLs.

Reply:
+1 agree, lot cleaner.